opm/simulators/linalg/PreconditionerFactory_impl.hpp

```diff
@@ -186,12 +194,23 @@ struct StandardPreconditioners
         // with the AMG hierarchy construction.
         if constexpr (std::is_same_v<O, Dune::OverlappingSchwarzOperator<M, V, V, C>>) {
           F::addCreator("amg", [](const O& op, const P& prm, const std::function<V()>&, std::size_t, const C& comm) {
+            using PrecPtr = std::shared_ptr<Dune::PreconditionerWithUpdate<V, V>>;
```

Comment:
I think PrecPtr is already defined in this class, no need to make this alias here?

Reply:
I don't think it is defined in the StandardPreconditioners struct. At least it doesn't compile when I don't define this alias.

Reply:
You are correct, I forgot that this was moved to the StandardPreconditioner helper. No need to change this.